lib/remote/services/auth_service.chopper.dart

```diff
@@ -6,6 +6,7 @@ part of 'auth_service.dart';
 // ChopperGenerator
 // **************************************************************************
 
+// ignore_for_file: always_put_control_body_on_new_line, always_specify_types, prefer_const_declarations
```

Comment:
Please don't include generated files under git.

Reply:
I approved because it's a small change but please do not forget about it.

Reply:
We haven't made those changes in the documentation though. So I don't think we can remove it yet in case a new contributor comes...
What do you think @bartekpacia ?

Reply:
@bartekpacia when you need a change request you might as well, request change. because once @Techno-Disaster changes, you review will be dismissed